docs/releases.md

```diff
@@ -0,0 +1,551 @@
+Please follow individual releases for more information.
+
+# 0.14.0
+
+## Migrating to 0.14.0
+
+### Breaking Changes
+
+Graphback 0.14 contains a lot of breaking changes that will improve and simplify your application code.
+
+#### CRUD Mutation return types are nullable
+
+Previously, the CRUD mutation resolvers did not allow returning null. These can now be null.
+
+```patch
+- createNote(input: CreateNoteInput!): Note!
++ createNote(input: CreateNoteInput!): Note
+```
+
+#### Changed annotation syntax
+
+All annotations now use a uniform syntax format: `@annotation(foo 'bar')` that is similar to GraphQL Directives
+
+Examples:
+
+```
+@oneToMany field: 'note'
+```
+
+becomes: 
+
+```
+@oneToMany(field: 'note')
+```
+
+```
+@db.type: 'timestamp'
+```
+
+becomes:
+
+```
+@db(type: 'timestamp')
+```
+
+#### Removed `@crud` annotation
+
+The `@crud` model annotation has been removed. To specify CRUD configuration on your model use the `@model` annotation.
+
+```
+"""
+@model
+@crud.delete: false
+@crud.update: true
+"""
+```
+
+becomes:
+
+```
+"""
+@model(delete: false, update: true)
+"""
+```
+
+### @graphback/runtime package is removed
+
+Runtime package should no longer be used in top level API.
+All functionalities were moved to the @graphback/core package.
+
+> NOTE: Users should never have runtime or core packages imported in their 
+applications and interact with graphback package as aggregator
+
+#### API code generation is no longer supported
+
+In 0.14.0 Graphback now supports runtime generation of your schema, resolvers and services. 
+
+Please manually remove your generated schema and resolver files.
+
+Remove `@graphback/codegen-resolvers` from your `package.json`.
+
+To use our runtime API, refer to the [Runtime API](./intro/serverless) documentation.
+
+#### New runtime API
+
+We have removed the complicated multi-step runtime API so that you can create your Graphback API in a couple of lines.
+
+If you were already using the runtime API, replace this:
+
+```ts
+const runtimeEngine = new GraphbackRuntime(model, graphbackConfig);
+const models = runtimeEngine.getDataSourceModels();
+const services = createKnexPGCRUDRuntimeServices(models, model, db, pubSub);
+const runtime = runtimeEngine.buildRuntime(services);
+```
+
+With this:
+
+```ts
+import { buildGraphbackAPI } from 'graphback'
+import { createKnexDbProvider, createCRUDService } from '@graphback/runtime-knex'
+import { PubSub } from 'graphql-subscriptions'
+
+const { typeDefs, resolvers, contextCreator } = buildGraphbackAPI(modelDefs, {
+  serviceCreator: createCRUDService({
+    pubSub: new PubSub()
+  }),
+  dataProviderCreator: createKnexDbProvider(dbConfig)
+});
+```
+
+For more advanced usage, refer to the [Runtime API](./intro/serverless) documentation.
+
+#### Client code generation now generates all GraphQL documents into one file
+
+1. Change your client output format from `gqlwithfragment` to `graphql`
```

Comment:
This can be reviewed since we are actually dropping the `format` option in this PR.